src/signals/incident/components/form/CheckboxInput/CheckboxInput.js

```diff
@@ -0,0 +1,94 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import map from 'lodash.map';
+import isObject from 'lodash.isobject';
+import { Label } from '@amsterdam/asc-ui';
+
+import Checkbox from 'components/Checkbox';
+
+import Header from '../Header';
+import styled from 'styled-components';
+
+function updateIncidentCheckboxMulti(checked, value, key, oldValue, meta, parent) {
+  let output = [...oldValue];
+  if (checked) {
+    output.push({
+      id: key,
+      label: value,
+    });
+  } else {
+    output = output.filter(item => item.id !== key);
+  }
+
+  parent.meta.updateIncident({ [meta.name]: output });
+}
+
+const CheckboxWrapper = styled.div`
+  & > :first-child {
+    margin-top: -6px; /* This margin enforces the 12px design distance from the title description */
+  }
+`;
+
+const CheckboxGroup = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const CheckboxInput = ({ handler, touched, hasError, meta, parent, getError, validatorsOrOpts }) =>
+  meta.isVisible && (
+    <Header meta={meta} options={validatorsOrOpts} touched={touched} hasError={hasError} getError={getError}>
+      <CheckboxWrapper>
+        {isObject(meta.values) ? (
+          <CheckboxGroup role="group">
```

Comment:
The label in the Header now applies a "htmlFor" property, but the id to which it points is not set. 

```suggestion
          <CheckboxGroup role="group" id={meta.name}>
```